pkgs/tools/audio/rescrobbled/default.nix

```
@@ -0,0 +1,49 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, pkg-config
+, dbus
+, openssl
+, runtimeShell
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "rescrobbled";
+  version = "0.4.0";
+
+  src = fetchFromGitHub {
+    owner = "InputUsername";
+    repo = "rescrobbled";
+    rev = "v${version}";
+    sha256 = "stz0mrXEFDGx+XLxMttb2XXR5n7cpw86USZQiUaagSw=";
+  };
+
+  cargoSha256 = "IxlhLIIHvEFBd+PghLsRq4PIxFwnHBJd0+79pxelzi4=";
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  buildInputs = [
+    dbus
+    openssl
+  ];
+
+  postPatch = ''
+    # Required for tests.
+    substituteInPlace src/filter.rs --replace '#!/usr/bin/bash' '#!${runtimeShell}'
```

Using runtimeShell in tests feels a bit awkward.. I don't think it's very important but maybe `stdenv.shell` or just plain `${bash}/bin/bahs`

---

agreed this should be
```suggestion
    substituteInPlace src/filter.rs --replace '#!/usr/bin/bash' '#!${bash}/bin/bash'
```